x/auth/client/cli/tx_sign.go

```diff
@@ -120,6 +120,7 @@ func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
 			txBuilder := txCfg.NewTxBuilder()
 			msgs := make([]sdk.Msg, 0)
 			newGasLimit := uint64(0)
+			feeAmount := sdk.Coins{}
```

Comment:
nit: `var feeAmount sdk.Coins`

Reply:
yes, happy to add them, I couldn't find any test file were `makeSignBatchCmd` (or similar) is being tested.
Should I create `auth/cli/tx_sign_test.go` ?